ios/Classes/ReactiveBle/Tasks/PeripheralTaskRegistry/PeripheralTaskRegistry.swift

```diff
@@ -2,21 +2,21 @@ import Foundation
 
 final class PeripheralTaskRegistry<Controller: PeripheralTaskController> {
 
-    typealias Task = Controller.Task
-    typealias TaskCompletionHandler = (Task.Key, Task.Params, Task.Result) -> Void
+    typealias CTask = Controller.SubjectTask
```

Comment:
`CTask` is no better than `PTask` was, so I suggest using that `SubjectTask` name here as well. Or I can take it along with the force-unwrapping question if/when I have time to.